xbmc/input/ButtonTranslator.cpp

```diff
@@ -1170,6 +1170,12 @@ int CButtonTranslator::GetFallbackWindow(int windowID)
   return -1;
 }
 
+CAction CButtonTranslator::GetGlobalAction(const CKey &key) const
+{
+  std::string strAction;
+  return GetActionCode(-1, key, strAction);
```

Comment:
You certainly mean this (https://github.com/xbmc/xbmc/blob/master/xbmc/input/ButtonTranslator.h#L116), right?